pkg/tenant/ensign_test.go

```diff
@@ -417,165 +403,3 @@ func (s *tenantTestSuite) TestTopicSubscriberBadEvents() {
 	s.subscriber.Stop()
 	sub.Wait()
 }
-
-func (s *tenantTestSuite) TestTopicSubscriberWaitForReady() {
```

Comment:
This test isn't relevant anymore, because `WaitForReady` was moved to Tenant server setup.